pkg/api/mlflow/dao/repositories/param.go

```diff
@@ -31,8 +32,50 @@ func NewParamRepository(db *gorm.DB) *ParamRepository {
 
 // CreateBatch creates []models.Param entities in batch.
 func (r ParamRepository) CreateBatch(ctx context.Context, batchSize int, params []models.Param) error {
-	if err := r.db.CreateInBatches(params, batchSize).Error; err != nil {
+	var err error
+	err = r.db.CreateInBatches(params, batchSize).Error
+	if err != nil {
```

Comment:
this block looks not well understandable.

Reply:
let's try to use more plain constructs. now it is a bit hard to understand what is going on when you read. which error belongs to which block. maybe we can change this blocks so the could be in more plain way? without nesting?

Reply:
why we can't before clean? and then just create?